dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/SymbolsInFile.java

```diff
@@ -20,21 +20,34 @@
 import com.google.common.collect.ImmutableSet;
 
 /**
- * A table of symbolic references and defined classes in a jar file or class file. The
- * symbolic references are from constant pool of class files.
+ * Symbolic references and defined classes in a file (jar file or class file). The
```

Comment:
This could be used to hold references from more than one file, right? There's nothing here that specifies a file. It's just a collection of class names and symbol references. And nothing in this class actually constrains these to come from constant pools. Let's just say this is a collection of symbol references and classes? (I'm still wondering whether those two things belong in the same place, but that can wait.)